src/interfaces/oracles/IDenominatedOracle.sol

```diff
@@ -8,6 +8,8 @@ interface IDenominatedOracle is IBaseOracle {
 
   /// @notice Throws if either the provided price source or denominated price source are null
   error DenominatedOracle_NullPriceSource();
+  /// @notice Throws if the feed price is invalid.
+  error DenominatedOracle_InvalidPriceFeed();
```

Comment:
`IBaseOracle` already brings in such error. 😉

Reply:
I thought I checked but I must have missed it, I removed the new one